source/_integrations/esphome.markdown

```diff
@@ -12,14 +12,18 @@ ha_codeowners:
 ha_domain: esphome
 ---
 
-This integration allows you to connect your [ESPHome](https://esphome.io) devices directly to Home Assistant with the [native ESPHome API](https://esphome.io/components/api.html).
+This integration allows [ESPHome](https://esphome.io) devices to connect directly to Home Assistant with the [native ESPHome API](https://esphome.io/components/api.html).
 
-## Set up the integration via the integrations screen
+## Automatic Configuration
 
-Menu: *Configuration* > *Integrations*
+An ESPHome device can be set up via the **Integrations** menu: **Configuration** > **Integrations**:
 
-Press on **ESPHome** and configure the integration:
+* Click the orange `+` on the lower-right of the screen and select **ESPHome**
 
-* Enter the address and port of your node. For example, if the node is called `livingroom`, the address would be `livingroom.local` and the port number would be `6053` (default).
+* Complete the dialog box and click `SUBMIT`. 
+    * **Host**: If the device is called `livingroom` in ESPHome, the Hostname would be `livingroom.local`
+    * **Port**: The default port is `6053`
 
-Home Assistant will then try to connect to the device. If you have a password set, Home Assistant will also ask you for it. After that, all the entities you have configured on your ESPHome node will automatically appear in Home Assistant.
+**Note:** If the node has a password set, Home Assistant prompt for it. 
```

Comment:
This doesn't have to be a note. It can be part of the general instructions above. We should avoid writing "notes" as much as possible.